src/components/questionnaire/QuestionnaireList.tsx

```diff
@@ -1,30 +1,45 @@
 import { useNavigation } from '@react-navigation/native';
 import React from 'react';
-import { StyleSheet, View } from 'react-native';
+import { Alert, Platform, StyleSheet, ToastAndroid, View } from 'react-native';
+import { getAllQuestionnaireDataByCode } from '../../data/api/Questionnaire';
+import { questionnaire } from '../../data/localStorage/ParticipantCode';
```

Comment:
Dit is verwarrend, want hieronder is Questionnaire met een hoofdletter gedifineerd. Geen duidelijk onderscheid tussen beide variabele nu

Reply:
Ik heb de interface naam aangepast